src/commands/test.js

```diff
@@ -13,7 +13,7 @@ const test = (context) => {
     extraEnv.DETAILED_LOG_TO_STDOUT = 'true';
   }
 
-  if (process.version !== LAMBDA_VERSION) {
+  if ( !LAMBDA_VERSION.includes(process.version)) {
```

Comment:
This will also need to revert when update since we'll be back to a single version